docker/ubuntu/baseimage-20.04/Dockerfile

```diff
@@ -88,41 +87,71 @@ RUN chmod +x local/bin/shfmt
 
 # }}}
 
-# LUAROCKS. {{{
+# LUAJIT. {{{
 
-RUN git clone https://github.com/torch/luajit-rocks.git
-WORKDIR luajit-rocks
-RUN git checkout 2c7496b905f6f972673effda4884766433b7583b
-RUN echo $HOME
-RUN cmake . -DWITH_LUAJIT21=ON -DCMAKE_INSTALL_PREFIX="$HOME/local"
-RUN make install
+ARG LUAJIT_VERSION=2.1.0-beta3
```

Comment:
It's worth noting that that is just as old as the one you're replacing. I'd probably just use the same commit hash as in https://github.com/koreader/koreader-base/pull/1765

Reply:
I had a hard time finding a combination of luajit/lua-lanes that does not result in a crashing and/or hanging luacheck. And no combo can pass the lua-lanes testsuite…